recipes/kuliya/all/conanfile.py

```diff
@@ -0,0 +1,40 @@
+from conan import ConanFile
+from conan.tools.files import copy, get, download
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.50.0"
+
+
+class JsmnConan(ConanFile):
+    name = "kuliya"
+    description = "Algeria's college hierarchy dataset as a C library."
+    license = "MIT"
+    topics = ("dataset", "api", "dz", "header-only")
+    homepage = "https://github.com/dzcode-io/kuliya"
+    url = "https://github.com/conan-io/conan-center-index"
+    package_type = "header-library"
```

Comment:
Upstream's CMakeLists seems to imply it wants to compile this library against a few other ones as dependencies, but those do not show up here. Are those just for educational pourposes and are they not needed in reallity? Or does the library need them?

Reply:
This recipe has no dependencies, those other dependencies are just needed for the `build.c` which is intended to be used only on the project actions.

TL;DR
The `kuliya` recipe will only have `kuliya.h` and `data.h` files